### PR TITLE
Log Faceshift FPS after a reset

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by Atlante45** [comment]
Maybe you should move that behaviour to FaceTracker.
That way it would work for all face tracker by default.

**Comment by ctrlaltdavid** [comment]
Yeah, I should. I've got a couple of other tidy-ups to do too. This is just a quick check-in for Philip for now.

**Comment by hifi-gustavo** [comment]
Build console output for commit: a141ebbf934373ffcf245d2f2e70fe002b9eff47 available at: https://gist.github.com/6b685f44fa4a740c9427<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4693.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4693.dmg

**Comment by Atlante45** [comment]
Ok

File: interface/src/devices/Faceshift.cpp [hunk 1]
```diff
@@ -10,6 +10,7 @@
 //
 
 #include <QTimer>
+#include <QElapsedTimer>
```
Comment (Atlante45) [inline interface/src/devices/Faceshift.cpp]: I don't think you need that one.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e33f3c9d00319836b8d444432435a033ba363dd4 available at: https://gist.github.com/6b685f44fa4a740c9427<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4693.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4693.dmg

**Pull request merged**